en/lessons/basics/collections.md

```diff
@@ -56,7 +56,12 @@ iex> [1,2,2,3,2,3] -- [1,2,3,2]
 [2, 3]
 ```
 
-**Note:** List subtraction uses [strict comparison](../basics/#comparison) to match the values.
+**Note:** List subtraction uses [strict comparison](../basics/#comparison) to match the values. For example:
```

Comment:
@midnio this is great 👍 Could we bump this file version from `1.2.3` to `1.2.4`?